sonar-update-center-mojo/src/main/java/org/sonar/updatecenter/mojo/HttpDownloader.java

```diff
@@ -75,4 +78,24 @@ File downloadFile(URL fileURL, File toFile) {
     return toFile;
   }
 
+
+  boolean downloadUrlStillValid(URL fileURL) {
+    log.info("Verify download URL (" + fileURL + ") is still valid");
```

Comment:
I don't think a log at info level is relevant.
This is a check, we only care if it fails.
I would either remove it or change level to debug